src/dateUtils.js

```diff
@@ -0,0 +1,42 @@
+const { DateTime, Duration } = require('luxon');
+
+const toDate = val => {
+  if (DateTime.isDateTime(val)){
+    return val;
+  } else if (typeof val === 'number'){
+    return DateTime.fromMillis(val);
+  } else if (typeof val === 'object' && val.getMonth && typeof val.getMonth === 'function'){
```

Comment:
Why do we need to check `val.getMonth` here?
If it's undefined, `typeof val.getMonth` will give `undefined`.

Reply:
This check isn't necessary, an error usually results from accessing an attribute on undefined

Reply:
Updated